api/libproject/gcloudcontext.rb

```diff
@@ -0,0 +1,38 @@
+require_relative "../../libproject/utils/common"
+require "json"
+
+class GcloudContextV2
+  attr_reader :account, :project
+
+  def initialize(options_parser)
+    # We use both gcloud and gsutil commands for various tasks. While gcloud can take arguments,
+    # gsutil uses the current gcloud config, so we want to grab and verify the account from there.
+    # We do NOT grab the project from gcloud config since that can easily be set to something
+    # dangerous, like a production project. Instead, we always require that parameter explicitly.
+    common = Common.new
+    options_parser.add_option(
+      "--project [GOOGLE_PROJECT]",
+      lambda {|opts, v| opts.project = v},
+      "Google project to act on (e.g. all-of-us-workbench-test)"
+    )
+    options_parser.add_validator lambda {|opts| raise ArgumentError unless opts.project}
+    options_parser.parse.validate
+    @project = options_parser.opts.project
+    common.status "Reading glcoud configuration..."
+    configs = common.capture_stdout \
+        %W{docker-compose run --rm api gcloud --format=json config configurations list}
+    active_config = JSON.parse(configs).select{|x| x["is_active"]}.first
+    common.status "Using '#{active_config["name"]}' gcloud configuration"
+    @account = active_config["properties"]["core"]["account"]
+    common.status "  account: #{@account}"
+    unless @account
+      common.error "Account must be set in gcloud config." \
+          " See gcloud config configurations --help."
```

Comment:
You could instead of "see gcloud configurations" say "run gcloud auth login <ACCOUNT>@pmi-ops.org".

Reply:
Yeah, that's better.